### ISSUE TITLE
`background-transition` support for `revealjs`

### ISSUE THREAD

**Issue by rleyvasal** [quarto-dev/quarto-cli#871]
Is [background-transition](https://revealjs.com/#/16) supported in Quarto?

Looking at the Quarto [Docs](https://quarto.org/docs/presentations/revealjs/#color-backgrounds) I did not find any reference to `background-transition`

I have tried the following but it does not work. 
```
## Slide Backgrounds {background="#43464B" background-transition="zoom"}
```

**Comment by cderv** [comment]
This correctly will create a zoom effect on the background. Do you use a recent Quarto version ? 

````markdown
---
title: "My simple title"
format: revealjs
---

## Header

hello world

## Slide Backgrounds {background="#43464B" background-transition="zoom"}

Content
````

This is documented here, but example shows slide content transition only https://quarto.org/docs/presentations/revealjs/advanced.html and also mention in demo slides https://quarto.org/docs/presentations/revealjs/demo/index.html#/slide-backgrounds

**Comment by rleyvasal** [comment]
thanks @cderv , your example works.

I found the problem happens when using a custom plugin (I am guessing the script is not playing well with background transition) 

I am not sure if  this is a problem with the custom plugin I used or if this problem happens with any custom plugins.
Note: you help me to get the process to install custom plugins on [issue #806](https://github.com/quarto-dev/quarto-cli/issues/806#issuecomment-1119420741)

```yaml
format: 
  revealjs:
    revealjs-plugins:
      - fullscreen
```


Feel free to close this if you think this is not an issue.
